combat-trainer.lic

```diff
@@ -1450,22 +1450,12 @@ class AbilityProcess
       .select { |name| Flags["ap-#{name}-expired"] }
```

Comment:
Should line 1442 use the new Khri helper in `common-arcana`?

Reply:
Can you activate a Khri that is already active? It uses a timer system there, which is why I didn't push it through. I think the Khri system might need a barb style setup. I think that's outside the scope of this PR. It can definitely get done though.

Reply:
Yeah, that does seem outside the scope of this PR.